source/api_documentation/network_integration/ringpools/index.rst

```diff
@@ -227,17 +227,27 @@ Content Type: application/json
 
       A filled field determines the center to start looking for local numbers at.
 
-      **"tn_prefix"**: can be either npa or npa or npa-nxx
+      **"tn_prefix"**: an npa
 
       **"zipcode"**: a valid zipcode
 
       **"latitude"**: a valid latitude
 
       **"longitude"**: a valid longitude
 
+  * - local_number_preferences
+    - hash
+    - All are optional
+
+      **"search_radius_miles"**: a number
```

Comment:
Can we provide more info to the user here? 0 is acceptable and that it's the distance from the center of their search location. The default is <= 240. It must be an integer.